tests/vspec/test_node_removal/test_files/units.yaml

```diff
@@ -0,0 +1,22 @@
+# Taken from the units.yaml of VSS catalog
```

Comment:
Interesting thought. Would we (should we) support unit deletion by the addition of the element marking ndoes as deleted?

Reply:
Can you elaborate @ppb2020 ?

Reply:
Basically, in the same manner that we can mark signal nodes as deleted, would it be useful to support the ability to delete unused units?

Not clear who would use this or when but, in the context of some implementation downloading the units file to a runtime, we could have the vss-tools generate a JSO, IDL, etc. version of the file, from which the marked as deleted units would be deleted.

Just some "wild" thoughts...